### ISSUE TITLE
Reading mode

### ISSUE THREAD

**Issue by VladThodo** [VladThodo/behe-explorer#15]
This browser needs a reading mode,and I'm curently implementing it,but any help is apreciated

**Comment by CrimsonFork** [comment]
Maybe you can add support for  [Red Moon](https://github.com/raatmarien/red-moon)?

**Comment by VladThodo** [comment]
This issue is pretty outdated as BeHe already has a reading mode ( available in the Pro version, but you can compile the source from GitHub, which will give you all the features of the pro).

What do you mean by Red Moon support?  Do you want Red Moon integration in BeHe's reading mode?

**Comment by CrimsonFork** [comment]
Yes, I'd like a second button near the color button thatbactivates Red Moon, or maybe that the color button does both.

**Comment by VladThodo** [comment]
I'd rather integrate Red Moon straight into BeHe, as I can see that Red Moon is open-source. So that you have a button that will display a dialog that will let you pick the intensity, color temperature, etc.

**Comment by CrimsonFork** [comment]
I just thought it would be easier to ensure the compatibility, rather than implement it partially or completely.

Am 8. Juni 2017 10:25:13 MESZ, schrieb Vlad Todosin <notifications@github.com>:
>I'd rather integrate Red Moon straight into BeHe, as I can see that Red
>Moon is open-source. So that you have a button that will display a
>dialog that will let you pick the intensity, color temperature, etc. 
>
>-- 
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/VladThodo/behe-explorer/issues/15#issuecomment-307035052

-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9/p≡p gesendet.

**Comment by VladThodo** [comment]
I understand it. I'll make a few tests to see what is the best way to implement it.

Thanks for getting involved :)

**Comment by CrimsonFork** [comment]
: )

Am 8. Juni 2017 18:56:26 MESZ, schrieb Vlad Todosin <notifications@github.com>:
>I undeestand it. I'll make a few tests to see what is the best way to
>implement it.
>
>Thanks for getting involved :)
>
>
>-- 
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/VladThodo/behe-explorer/issues/15#issuecomment-307163851

-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9/p≡p gesendet.
